Products/ZPsycopgDA/dtml/add.dtml

```diff
@@ -62,14 +59,14 @@
     <div class="form-group row">
       <label for="check" class="form-label col-sm-4 col-md-3">Connect immediately</label>
       <div class="col-sm-5 col-md-1">
-        <input class="btn btn-primary" type="checkbox" id="check" name="check" value="YES" checked="YES" class="form-control"/>
+        <input class="btn btn-primary" type="checkbox" id="check" name="check" value="YES" checked="checked" />
```

Comment:
Should the `form-control` class be added to the actual class attribute instead?

Reply:
form-control doesn't change a lot. Bootstrap says to use "form-check-input" to style check boxes, but that isn't used anywhere in the ZMI either.

Also, as above, not my idea originally.

Reply:
Doing a code scan on Zope4, I see that "form-check-input" is used as a standard. Will add that.

The "btn btn-primary" styling is not used for checkboxes at all. That will go.